apps/ledger-live-mobile/ios/ledgerlivemobile/Info.plist

```diff
@@ -135,4 +135,4 @@
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
 </dict>
-</plist>
+</plist>
```

Comment:
Could you remove everything that is not related to the Icon integration ? :pray: